json-endpoints/class.wpcom-json-api-get-comments-tree-endpoint.php

```diff
@@ -87,10 +90,10 @@ function callback( $path = '', $blog_id = 0 ) {
 		}
 
 		$args = $this->query_args();
-		$comment_status = $args['status'];
+		$comment_status = empty( $args['status'] ) ? 'all' : $args['status'];
 
 		if ( ! $this->validate_status_param( $comment_status ) ) {
-			return new WP_Error( 'invalid_status', 'Invalid comment status value provided ', 400 );
+			return new WP_Error( 'invalid_status', "Invalid comment status value provided: '$comment_status''.", 400 );
```

Comment:
Sorry, but it looks like there's an extra single quote here: ` '$comment_status''`
Otherwise things look good!